#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_configure.in_added-LIBS.dpatch by Frederik Dannemare <frederik@dannemare.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add -lvorbis -lvorbisenc -ldts -la52 to LIBS

@DPATCH@

--- motion-3.1.19/configure.in.orig	2005-03-14 12:09:58.712691480 +0100
+++ motion-3.1.19/configure.in	2005-03-14 12:12:43.752601600 +0100
@@ -144,7 +144,7 @@
 		echo ""
 	fi
 	CFLAGS="${CFLAGS} -DHAVE_FFMPEG"
-	LIBS="$LIBS -L${FFMPEG} -lavformat -lavcodec -lm -lz"
+	LIBS="$LIBS -L${FFMPEG} -lavformat -lavcodec -lvorbis -lvorbisenc -ldts -la52 -lm -lz"
 	FFMPEG_OBJ="ffmpeg.o"
 	AC_SUBST(FFMPEG_OBJ)
 	LDFLAGS="-Wl,-rpath,${FFMPEG}"
